torch/csrc/lazy/ts_backend/ops/device_data.cpp

```diff
@@ -12,7 +12,7 @@ DeviceData::DeviceData(std::shared_ptr<BackendData> data)
           ltc_device_data,
           data->shape(),
           /*num_outputs=*/1,
-          /*hash_seed=*/static_cast<uint32_t>(101)),
+          /*hash_seed=*/Node::NextNodeListIndex()),
```

Comment:
hmm.. we're saying we want the DeviceData node hash to be sensitive to whether it was used earlier or later in the trace, in addition to tensor shape/dtype.

i think it is correct, but i'm not sure if it helps reuse?